spec/src/main/asciidoc/changelog.asciidoc

```diff
@@ -19,6 +19,16 @@
 
 :numbered!:
 
+=== Release 1.2.1
```

Comment:
We should decide what version to release whether 1.2.1 or 1.3.

There are no API breaking changes. The only breaking changes are 
*  Renamed TCK class (https://github.com/eclipse/microprofile-opentracing/pull/106[#106])
* Slightly changed generated span data - added HTTP method to http-path operation name (https://github.com/eclipse/microprofile-opentracing/pull/125[#125]

I think we could go with 1.2.1

cc @pilhuhn @kwsutter

Reply:
My vote is 1.2.1